geometry_msgs/PointStamped centroid
float32 variance
uint32 samples
